### ISSUE TITLE
Blast and Menting

### ISSUE THREAD

**Issue by thebastien007** [kicjow/Crazy-Enchantments#180]
Hey,
Would it be possible to add an option for deactivate the enchantment Blast on the pickaxe with Menting ? Because it's very cheat if we have a pickaxe with Menting and Blast...
It would be really great!
Thank's for reading
Bastien

Ps: Sorry for my bad english =(

**Comment by kicjow** [comment]
I understand why you would want that but the thing is that's not much different then the player using mending and mining all that out instead. That's why I don't think ill do that. Blast is basically a faster way of mining.

**Comment by thebastien007** [comment]
Yes but l'enchant Blast is very cheat with the economy of server. And, if there is mending on a pickaxe, then a pickaxe is unbreaking. So the players buy Blast only 1 time.
Can you really not put this option ?
I'm seriously bothering.....

**Comment by kicjow** [comment]
I might try and do something about it but I am not sure if I will sorry.
